### PR TITLE
[new] Add coolprop [re-fork]

### PR DESCRIPTION
**Pull request by doomiris**
Pull request for issue: #13317

## Profile of the lib

* Git repository (required): https://github.com/CoolProp/CoolProp.git
* Official website (optional, not the repository): http://coolprop.sourceforge.net/
* NPM package url (optional): none
* License and its reference: MIT
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 47
  * Count of stars: 236
  * Count of forks: 115
* NPM download stats (optional): none
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by doomiris** [comment]
@PeterDaveHello @ryankirkman Please review, thanks a lot.

File: ajax/libs/coolprop/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "source": "git",
+    "target": "git://github.com/CoolProp/CoolProp.git",
+    "fileMap": [
+      {
+        "basePath": "wrappers/Javascript",
+        "files": [
+          "coolprop.js",
+          "coolprop.min.js",
+          "coolprop.wasm"
+        ]
```
Comment (MattIPv4) [inline ajax/libs/coolprop/package.json]: These files do not appear in the latest tagged release, 6.3.0: https://github.com/CoolProp/CoolProp/tree/v6.3.0/wrappers/Javascript

File: ajax/libs/coolprop/6.3.0/coolprop.js [hunk 2]
```diff
@@ -0,0 +1,10661 @@
+// Copyright 2010 The Emscripten Authors.  All rights reserved.
```
Comment (MattIPv4) [inline ajax/libs/coolprop/6.3.0/coolprop.js]: This file does not appear in the latest tagged release, 6.3.0: https://github.com/CoolProp/CoolProp/tree/v6.3.0/wrappers/Javascript

File: ajax/libs/coolprop/6.3.0/coolprop.min.js [hunk 3]
```diff
@@ -0,0 +1 @@
...!0}).node,n=FS.lookupPath(i,{parent:!0}).node}catch(e){throw new FS.ErrnoError(ERRNO_CODES.EBUSY)}if(!t||!n)throw new FS.ErrnoError(ERRNO_CODES.ENOENT);if(t.mount!==n.mount)throw new FS.ErrnoError(ERRNO_CODES.EXDEV);var u,d=FS.lookupNode(t,o),s=PATH.relative(e,a);if("."!==s.charAt(0))throw new FS.ErrnoError(ERRNO_CODES.EINVAL);if("."!==(s=PATH.relative(i,r)).charAt(0))throw new FS.ErrnoError(ERRNO_CODES.ENOTEMPTY);try{u=FS.lookupNode(n,l)}catch(e){}if(d!==u){var c=FS.isDir(d.mode),_=FS.mayDelete(t,o,c);if(_)throw new FS.ErrnoError(_);if(_=u?FS.mayDelete(n,l,c):FS.mayCreate(n,l))throw new FS.ErrnoError(_);if(!t.node_ops.rename)throw new FS.ErrnoError(ERRNO_CODES.EPERM);if(FS.isMountpoint(d)||u&&FS.isMountpoint(u))throw new FS.ErrnoError(ERRNO_CODES.EBUSY);if(n!==t&&(_=FS.nodePermissions(t,"w")))throw new FS.ErrnoError(_);try{FS.trackingDelegate.willMovePath&&FS.trackingDelegate.willMovePath(e,i)}catch(t){console.log("FS.trackingDelegate['willMovePath']('"+e+"', '"+i+"') threw an exception: "+t.message)}FS.hashRemoveNode(d);try{t.node_ops.rename(d,n,l)}catch(e){throw e}finally{FS.hashAddNode(d)}try{FS.trackingDelegate.onMovePath&&FS.trackingDelegate.onMovePath(e,i)}catch(t){console.l
```
Comment (MattIPv4) [inline ajax/libs/coolprop/6.3.0/coolprop.min.js]: This file does not appear in the latest tagged release, 6.3.0: https://github.com/CoolProp/CoolProp/tree/v6.3.0/wrappers/Javascript

File: ajax/libs/coolprop/package.json (hunk 1 above)
Comment (MattIPv4) [inline ajax/libs/coolprop/package.json]: These files still do not exist: https://github.com/CoolProp/CoolProp/tree/v6.3.0/wrappers/Javascript

We cannot have an auto-update config that doesn't actually bring in any files, as the library will then never get any files on cdnjs.

File: ajax/libs/coolprop/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "coolprop",
+  "description": "CoolProp is a thermophysical property database. It offers similar functionality to REFPROP",
+  "version": "6.3.0",
```
Comment (MattIPv4) [inline ajax/libs/coolprop/package.json]: Please remove the version attribute, the auto-update bot will populate this itself when it pulls in versioned files

**Comment by MattIPv4** [comment]
No progress, no further tags released in repo, files still aren't available in repo -- closing.

**Pull request closed without merging**